daemon/cmd/daemon_main.go

```diff
@@ -971,6 +971,10 @@ func init() {
 	flags.Bool(option.EnableCustomCallsName, false, "Enable tail call hooks for custom eBPF programs")
 	option.BindEnv(option.EnableCustomCallsName)
 
+	flags.Bool(option.NetfilterCompatibleMode, false, "If set to true, guarantees that traffic will pass through netfilter and that iptable rules will be enforced. This mode may reduce network throughput. If set to false (default), there are no guarantees: some traffic may skip netfilter.")
```

Comment:
Currently, it does not guarantee that _all_ traffic will pass through netfilter (https://github.com/cilium/cilium/issues/16166).

Reply:
As suggested, will fix in separate PR.